### PR TITLE
Added additional width to containers and breakpoints

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/scss/griidc-colors.scss [hunk 1]
```diff
@@ -103,3 +103,23 @@ $navbar-light-active-color: color('black');
 $navbar-light-hover-color: theme-color('primary');
 
 $dropdown-link-hover-color: theme-color('primary');
+
```
Comment (mickel1138) [inline assets/scss/griidc-colors.scss:106]: Maybe one day we should move the common things at a higher level?

File: assets/js/vue/ResearchGroupApp.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="container-xl" ref="formContainer">
+    <div class="container" ref="formContainer">
```
Comment (mickel1138) [inline assets/js/vue/ResearchGroupApp.vue:2]: This makes the background narrower than before. The container is also really tight around the content. For example the cards, and the results per page selector. Maybe it should be wider, or need padding?

File: assets/scss/griidc-colors.scss (hunk 1 above)
Comment (praneethpr) [reply assets/scss/griidc-colors.scss:106]: Yes maybe we can have a common file for both themes

File: assets/js/vue/ResearchGroupApp.vue (hunk 2 above)
Comment (praneethpr) [reply assets/js/vue/ResearchGroupApp.vue:2]: Sorry, i dint notice the background color thing. I fixed it, not sure about the content being tight. I haven't changed any of that, and one of the things in the ticket was to reduce padding

Comment (fightingtexasaggie) [reply assets/js/vue/ResearchGroupApp.vue:2]: I'm not seeing where this is really tight around content. It looks fine in both Firefox and Chrome to me on the search page.

File: assets/js/vue/components/search/SearchForm.vue [hunk 3]
```diff
@@ -66,31 +66,31 @@
                         </div>
                     </b-form>
                 </div>
-            </div>
```
Comment (fightingtexasaggie) [inline assets/js/vue/components/search/SearchForm.vue:69]: Where's this div open?

File: assets/js/vue/ResearchGroupApp.vue (hunk 2 above)
Comment (mickel1138) [reply assets/js/vue/ResearchGroupApp.vue:2]: > I'm not seeing where this is really tight around content. It looks fine in both Firefox and Chrome to me on the search page.

That's been fixed now.

File: assets/js/vue/components/search/SearchForm.vue (hunk 3 above)
Comment (praneethpr) [reply assets/js/vue/components/search/SearchForm.vue:69]: I cross checked, i don't see any orphan div

Comment (fightingtexasaggie) [reply assets/js/vue/components/search/SearchForm.vue:69]: just odd how github displayed it I guess

**Pull request merged**